app/api/tickets.py

```diff
@@ -247,6 +247,12 @@ def before_update_object(self, ticket, data, view_kwargs):
                 "Can't delete a ticket that has sales",
             )
 
+        if data.get('event') and data.get('sales_ends_at') > event.ends_at:
+            raise UnprocessableEntityError(
+                {'sales_ends_at': '/data/attributes/sales-ends-at'},
+                "End of ticket sales date of "+data.get('name')+" cannot be after end of event date.",
```

Comment:
```suggestion
                f'End of ticket sales date of "{ data.get('name') }" cannot be after end of event date',
```

Reply:
> f'End of ticket sales date of "{ data.get('name') }" cannot be after end of event date',

this is giving error.